Sources/KeyboardSupport/KeyboardScrollable.swift

```diff
@@ -22,7 +22,7 @@ public struct KeyboardInfo {
     public let finalFrame: CGRect
     public let animationDuration: TimeInterval
     public let animationCurve: UInt
-    
+
```

Comment:
Just to keep things tidy, could you revert all of these whitespace changes?

Reply:
Gotcha, I thought removing the whitespace would be more tidy, not less!

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/61/commits/e3ba7670babb46efdc74bbe985614efd1768432b